docs/root/intro/arch_overview/observability/access_logging.rst

```diff
@@ -17,29 +17,54 @@ logs<envoy_v3_api_field_config.listener.v3.Listener.access_log>`. The listener a
 HTTP request access logging and can be enabled separately and independently from
 filter access logs.
 
+If access log is enabled, then by default it will be reported to the configured sinks at the end of a TCP
```

Comment:
should we update this text to remove references to TCP which is now no longer implemented?

Reply:
Not sure what you mean, what is no longer implemented in TCP?

Reply:
ah apologies, for some reason I thought the TCP log stream info were done in the same PR as the tcp log flushing.  the suggestion was in error and docs are fine as is.